deploy/create_linux_appimage.sh

```diff
@@ -58,7 +56,7 @@ cp -L libdirectfb/usr/lib/x86_64-linux-gnu/libfusion-1.2.so.9 ${APPDIR}/usr/lib/
 cp -L libdirectfb/usr/lib/x86_64-linux-gnu/libdirect-1.2.so.9 ${APPDIR}/usr/lib/x86_64-linux-gnu/
 
 # copy libts-0.0-0
-wget -c --quiet http://ftp.us.debian.org/debian/pool/main/t/tslib/libts-0.0-0_1.0-11_amd64.deb
+wget -c --quiet http://old-releases.ubuntu.com/ubuntu/pool/universe/t/tslib/libts-0.0-0_1.0-11_amd64.deb
```

Comment:
I wonder if this should be removed. From the Issue comments if I remember it has something to do with touchscreens. Not sure why it would be QGCs job to install that. If a Linux user was using a touchscreen wouldn't they need to install this themselves to get it to work generically?

Reply:
The following is an excerpt of `apt-cache show libts-dev`:

Homepage: http://tslib.org
Description-en: touch screen library, development files
 Tslib is an abstraction layer for touchscreen panel events, as well as a
 filter stack for the manipulation of those events.
 
 So, yes, it is specific to touch screens.
 
 Regarding the idea of just removing that line from `create_linux_appimage.sh`, I would tend to agree.
 
 Since no QGroundcontrol AppImage has actually contained libts for many releases, that seems like enough evidence that it wasn't really needed.  I would tend to say "remove it and wait til anyone complains."
 
 Corollary: if someone does complain and some variation of this `wget libts` gets re-added to the script, then at least you can then capture better documentation about the rationale.
 
 It seems that libts was added back in 2016, in case this helps jog anyone's memory: https://github.com/mavlink/qgroundcontrol/commit/78d203bedf758462

Reply:
Can you remove then. 

Also if I remember, udev is maybe related to flashing. Can you verify that flashing still works on a machine without it? Sorry but Linux is a foreign animal to me so I'm not super helpful on understanding the exact details of why these things may be needed. My concern would be that most Linux users may also have installed the PX4 firmware build pre-reqs which I think installs udev (which is how my VM is set up). So for them it may work. But for someone with a clean Linux it may not.

Reply:
I will remove the libts line right away.

Regarding the rest:

quoting:

```
libudev.h provides APIs to introspect and enumerate devices on the local system.
```

from here: https://www.freedesktop.org/software/systemd/man/libudev.html

Labeling *my* weaknesses: Windows is a (mostly) foreign animal to *me*, so I am 100% empathetic to your situation. I'm happy to work in a bi-partisan way with you "across the OS aisle" ;) 

I think the odds of libudev not being on a modern machine used for "Desktop" usage is very low.

I just checked the reverse dependencies (things that depend on libudev) using `apt-cache rdepends libudev1`, and I see that all these would bring udev with them (or else not work):

```
...
  network-manager
  libudev-dev
  libnm0
  libnm-glib4
  libgnome-desktop-3-17
  libgnome-bluetooth13
  libd3dadapter9-mesa-dev
  libd3dadapter9-mesa
  initramfs-tools-bin
  xserver-xorg-core
  pulseaudio
  bluez
  xserver-xorg-input-xwiimote
  xserver-xorg-input-evdev
  vlc-plugin-base
  ubuntu-core-libs
  scanbd
  ring-daemon
  libgnome-desktop-3-17
  libgnome-bluetooth13
  libdevmapper1.02.1
  libclutter-1.0-0
...
```

... among many others.

I think there need **not** be concern for libudev being missing. However, the other feature of using an `AppImage` (as you know) is to include a particular **version** of the library.  So I suppose that could still be an issue.

After I opened this pull request, I subsequently read through a small handful of bug reports on QGroundcontrol related to gstreamer, and to the discussion/debate surrounding inclusion of gstreamer in the AppImage.

I would recommend that any concerns about libudev be bundled into the general debate on the AppImage issues. At some point it seems clear that someone will need to "own" and sort through all of that.

For this (small) PR, I'm merely suggesting to remove lines from a script where all the involved lines have failed for some time.  

Whether (and how) to bundle these failing-for-some-time libs in the future seems arguably out of scope. (I could argue it both ways, though.)

Reply:
I just checked a virgin Ubuntu 18.04 and it has udev installed in it. So doesn't seem to be needed with that either. So seems perfectly fine to not include.